src/styles/bootstrap.scss

```diff
@@ -142,7 +142,30 @@
     /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
     @extend .form-control;
 
-    width: unset;
+    width: 98%;
+    border: 1px solid #424242;
+    border-radius: 5px;
+    box-shadow: 1px 5px 5px -3px #808080;
+  }
+
+  .os-form-control:focus {
+    border-color: #159eb2;
+    box-shadow: 1px 5px 5px -3px #808080;
+
+  }
+
+  .os-form-control-wrong-answer-choice {
+    border-color: #fc0516;
+  }
+
+
+
+  .os-form-control-correct-answer-choice {
+    border-color: #019920;
+  }
+
+  .os-form-control:disabled {
+    box-shadow: none;
```

Comment:
A couple of general comments on these styles:

1. We may not want to hard-code some of these constant values and instead use / define variables like we do [here](https://github.com/openstax/k12-apps-raise/blob/ab10977c533cf119e74c3caf207b084d9c1c8b82/src/styles/interactives.scss#L1-L8)
2. I feel like there may be some redundancy with some of these classes being added in `bootstrap.scss` and some of the ones we're creating in `interactives.css` such as [here](https://github.com/openstax/k12-apps-raise/blob/ab10977c533cf119e74c3caf207b084d9c1c8b82/src/styles/interactives.scss#L34-L37). Perhaps this is a known thing that you and @MReyna12 have already discussed and plan to consolidate once everything is in the same branch, but just wanted to point it out / ask :man_shrugging: .

Reply:
Going to both points - there is likely a number of areas of redundancy. The intent was to comb through redundant CSS post-merge into the v2-ib-style-updates branch and consolidate then. There may be some redundancy in `bootstrap.scss` and `interactives.scss` due to specificity issues caused by the os-raise-bootstrap class.